porchlightapi/settings.py

```diff
@@ -35,5 +36,14 @@
                                'PORCHLIGHT_JSON_FILE',
                                PORCHLIGHT_JSON_FILE_DEFAULT)
 
+PORCHLIGHT_GITHUB_AUTH_DEFAULT = None
+PORCHLIGHT_GITHUB_AUTH = getattr(settings,
+                                 'PORCHLIGHT_GITHUB_AUTH',
+                                 PORCHLIGHT_GITHUB_AUTH_DEFAULT)
+
+PORCHLIGHT_GITHUB_TAG_PATTERN_DEFAULT = r'^v[.0-9]+'
```

Comment:
This seems like a perfectly reasonable approach.  We set a default and let others decide if they want to change it.  :+1: @willbarton